django/db/models/base.py

```diff
@@ -585,6 +586,17 @@ def from_db(cls, db, field_names, values):
         new._state.db = db
         return new
 
+    def to_dict(self):
+        model_dict = {}
+        for field in self._meta.fields:
+            field_name = field.name
+            field_value = getattr(self, field_name)
+            model_dict[field_name] = field_value
+        return model_dict
+
+    def to_json(self):
+        return json.dumps(self.to_dict())
+
```

Comment:
Thanks for this patch, however a similar functions already exist, e.g. `model_to_dict()`.